tools/c7n_oci/c7n_oci/utils.py

```diff
@@ -17,3 +17,7 @@ def join_output(output_dir, suffix):
         path=output_url_parts.path.rstrip("/") + "/%s" % suffix
     )
     return urlparse.urlunparse(output_url_parts)
+
+
+def spec_version():
```

Comment:
minor nit, but any reason on function vs treating this as a module level constant? there's no ability to update at runtime in python.